app/controllers/project.py

```diff
@@ -231,10 +265,21 @@ def get(self):
             try:
 
                 if (keywords == ''):
-
-                    pagination = Project.query.filter(or_(Project.owner_id == current_user_id, Project.users.any(
-                        ProjectUser.user_id == current_user_id))).order_by(Project.date_created.desc()).paginate(
-                        page=page, per_page=per_page, error_out=False)
+                    if attribute:
+                        paginated = (Project.query.filter(getattr(Project, attribute) == attribute_value).order_by(Project.date_created.desc()).paginate(page=page, per_page=per_page, error_out=False))
+                        projects = paginated.items
+                        pagination = {
+                            'total': paginated.total,
+                            'pages': paginated.pages,
+                            'page': paginated.page,
+                            'per_page': paginated.per_page,
+                            'next': paginated.next_num,
+                            'prev': paginated.prev_num
+                        }
+                    else:
+                        pagination = Project.query.filter(or_(Project.owner_id == current_user_id, Project.users.any(
```

Comment:
Pagination seems to fail (ie is empty) when those any of the filters is applied, as shown below
<img width="1431" alt="Screenshot 2023-08-18 at 1 22 16 AM" src="https://github.com/crane-cloud/backend/assets/18049550/2bde21e9-e92e-465a-a280-26b59e5fde71">

Reply:
let me look into this

Reply:
this has been fixed